test/trn_controller/controller.py

```diff
@@ -65,13 +65,13 @@ def create_network(self, vni, netid, cidr, switches):
         for s in switches:
             self.add_switch(vni, netid, s)
 
-    def create_simple_endpoint(self, vni, netid, ip, host):
+    def create_endpoint(self, vni, netid, ip, host, ep_type="simple"):
```

Comment:
Okay, I can revert the change for now. (It will create a merge conflict for my other PR too 😞)

However I think this refactor will make the code easier to maintain in the future.
As of now, when we add a new endpoint type, we will have to add functions in the 3 classes.

controller.py, vpc.py, and network.py
So with endpoint types: host, vxlan and simple, we have 9 functions to manage.

With the change, we only need to add another condition in the vpc create_endpoint function, and add another function in the network class.
With endpoint types: host, vxlan, and simple, we now have 5 functions to manage